### ISSUE TITLE
Application-specific key combination

### ISSUE THREAD

**Issue by OlegPQ** [koekeishiya/skhd#296]
I have the following simple combination that launches the calendar

`lalt - c : open -a "Calendar"`

and I am looking for a way to make it work only if I am in Firefox or Chrome browser. 
Is this possible? Maybe it can be done through osascript?

**Comment by noncog** [comment]
Yes. See the following lines in the example configuration for app specific keybinds: https://github.com/koekeishiya/skhd/blob/f88e7ad403ebbee1b8bac988d8b162d595f595c4/examples/skhdrc#L83C1-L91C4
